src/Features/CSharp/Portable/CodeFixes/SimplifyTypeNames/SimplifyTypeNamesCodeFixProvider.cs

```diff
@@ -96,6 +97,9 @@ internal static string GetCodeActionId(string diagnosticId, string nodeText)
                 case IDEDiagnosticIds.SimplifyThisOrMeDiagnosticId:
                     return CSharpFeaturesResources.SimplifyThisQualification;
```

Comment:
Rename to `RemoveThisQualification` to be consistent? (Note: that's what the resource actually is -- "Remove 'this' qualification")